recipes/cdp/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "cdp" %}
+{% set version = "1.3.3" %}
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/CDAT/cdp/archive/v{{ version }}.tar.gz
+  sha256: 8c3e69c695ecdf0c8e3d0fe38e4ae6987445aff104a89e837e496818ff9bb343
+
+build:
+    number: 0
+    noarch: python
+    script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
+
+requirements:
+  build:
+    - python {{ python }}
+    - setuptools
+    - pip
+  run:
+    - python {{ python }}
+    - output_viewer
+    - pyyaml
+    - dask
+    - distributed
+    - six
+
+test:
+  commands:
+      - python -c "import cdp"  
+
+about:
+  home: https://github.com/cdat/cdp
+  summary: The Community Diagnostics Package (CDP) is a framework for developing new climate diagnostics. CDP allows for a modular implementation of the components required for running diagnostics. The design of CDP consists of modules to handle the user-defined parameters, metrics, provenance, file I/O, output of results and algorithms for calculating the diagnostics.
+  license: BSD3
```

Comment:
I don't see any licensing info. Please clarify where the `BSD-3-Clause` is mentioned.